test/format/verify.c

```diff
@@ -273,7 +273,8 @@ table_verify_mirror(WT_CONNECTION *conn, TABLE *base, TABLE *table, const char *
             testutil_check(table_cursor->get_value(table_cursor, &table_value));
 
             if (base_keyno != table_keyno || base_value.size != table_value.size ||
-              memcmp(base_value.data, table_value.data, base_value.size) != 0) {
+              (table_value.size != 0 &&
```

Comment:
what happens if the base_value.size == 0?.
Basically why only checking table_value.size and ignoring base_value.size.

Reply:
This is because at this point we are sure that base_value.size and table_value.size are the same (second condition of the if statement).